packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -82,7 +92,9 @@ const convertRemoteParticipantsToParticipantListParticipants = (
             participant.isSpeaking,
             /* @conditional-compile-remove(raise-hand) */
             participant.raisedHand,
-            localUserCanRemoveOthers
+            localUserCanRemoveOthers,
+            /* @conditional-compile-remove(reaction) */
+            remoteParticipantReaction
```

Comment:
`memoizeFn` memoizes the result based on the in params.  However, in javascript this:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/d0731655-50d0-4697-aff2-8b6bb6618b17)
Will create a new object each time it runs, and objects are passed by reference when pushed to functions, so this will invalidate the memoize on each run.

Two alternative options:
1. (probably easier) Pass `participant.reactionState.reactionMessage.reactionType` and `participant.reactionState.receivedAt` into the memoize function and construct the new object there
2. use `memoizeFn` to first memozie the remoteParticipantReaction object and then pass that in

Reply:
In order to do minimal change and keep the public API design as it is, I have chose the 2nd approach. Thanks for this suggession.